ckan/lib/plugins.py

```diff
@@ -524,3 +524,31 @@ def activity_template(self):
         return 'organization/activity_stream.html'
 
 _default_organization_plugin = DefaultOrganizationForm()
+
+
+class DefaultTranslation(object):
+    def directory(self):
+        '''Change the directory of the *.mo translation files
+
+        The default implementation assumes the plugin is
+        ckanext/myplugin/plugin.py and the translations are stored in
+        i18n/
+        '''
+        import os
+        return os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
don't know if this is useful here https://github.com/open-data/ckanext-scheming/blob/master/ckanext/scheming/plugins.py#L301-L315

That's how I go from a python import path like ckanext.myplugin to a file system path